src/js/gi/components/content/VideoSidebar.jsx

```diff
@@ -11,6 +11,10 @@ class VideoSidebar extends React.Component {
         </p>
         <iframe width="100%" src="https://www.youtube.com/embed/Z1ttkv9oRI4"
             title="Know Before You Go" frameBorder="0" allowFullScreen></iframe>
+        <h5>Ready to apply?</h5>
+        <a href="https://www.vets.gov/education/apply/">
```

Comment:
Do you want to open this in a new tab? In case they are in the middle of using the tool?